### PR TITLE
Add Dockerfile and other changes for the new st2garbagecollector service

### PR DESCRIPTION
**Pull request by Kami**
The title says it all.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ docker build -t stackstorm/st2notifier:1.1.2 st2notifier/
 docker build -t stackstorm/st2resultstracker:1.1.2 st2resultstracker/
 docker build -t stackstorm/st2rulesengine:1.1.2 st2rulesengine/
 docker build -t stackstorm/st2sensorcontainer:1.1.2 st2sensorcontainer/
+docker build -t stackstorm/st2garbagecollector:1.1.2 st2garbagecollector/
```
Comment (armab) [inline README.md]: And push:
https://github.com/StackStorm/st2-dockerfiles/blob/new_garbage_collector_service/README.md#4-deploy-to-docker-hub

**Comment by dennybaa** [comment]
@Kami Thank you, we'll merge it as soon as the package is added. Huf, spinning up build time again(

**Comment by Kami** [comment]
@dennybaa This trend won't change - we will be adding more services in the future. So something we need to keep in mind and design / account for.

**Comment by lakshmi-kannan** [comment]
@dennybaa Do we need to branch v1.3 for this repo? How would this work when we want to generate 1.3 packages for st2? cc: @armab

**Comment by dennybaa** [comment]
Noop, we don't as we don't really need this repo as well. In my opinion this repo will be deprecated soon. So it's safe to merge, anyway we are not building any of this containers anymore.

**Comment by armab** [comment]
@enykeev has in plans to resurrect this repo to build and deploy Docker images.
We stopped doing that recently https://hub.docker.com/r/stackstorm/

From what I know, his overall approach is pretty elegant, so +:100:, let he decide what he needs, including branches, whatever.

**Pull request merged**